presto-cassandra/pom.xml

```diff
@@ -17,8 +17,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.facebook.presto.cassandra</groupId>
-            <artifactId>cassandra-driver</artifactId>
+            <groupId>com.datastax.cassandra</groupId>
```

Comment:
Can you put this in the dependency management section of the parent pom?

Reply:
done